velox/docs/monthly-updates/december-2022.rst

```diff
@@ -0,0 +1,66 @@
+********************
+December 2022 Update
+********************
+
+Core Library
+============
+
+* Simplify the memory management implementation by consolidating the memory
+  objects from previously 15 down to 8.
+
+Presto Functions
+================
+
+* Add :func:`array_has_duplicates`, :func:`crc32`, :func:`hmac_sha1`,
+  :func:`hmac_sha256`, :func:`hmac_sha512`, :func:`json_size` scalar functions.
+
+Spark Functions
+===============
+
+* Add shiftleft and shiftright scalar functions.
+
+Hive Connector
+==============
+
+* Add support for IS NULL and IS NOT NULL filter pushdown for maps and arrays
+  to DWRF Reader.
+* Add support for skipping files and row groups based on remaining filer to
+  DWRF Reader. :pr:`3549`
```

Comment:
This is common for both Parquet and DWRF, file skipping based on remaining filter is not implemented yet (only row groups for now), also there is a typo in "remaining filter"